Taarafo.Core.Tests.Unit/Services/Foundations/PostImpressions/PostImpressionServiceTests.Validations.RemoveById.cs

```diff
@@ -0,0 +1,115 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE TO CONNECT THE WORLD
+// ---------------------------------------------------------------
+
+using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Moq;
+using Taarafo.Core.Models.PostImpressions;
+using Taarafo.Core.Models.PostImpressions.Exceptions;
+using Xunit;
+
+namespace Taarafo.Core.Tests.Unit.Services.Foundations.PostImpressions
+{
+    public partial class PostImpressionServiceTests
+    {
+        [Fact]
+        public async Task ShouldThrowValidatonExceptionOnRemoveIfIdIsInvalidAndLogItAsync()
+        {
+            //given
+            PostImpression randomPostImpression = CreateRandomPostImpression();
```

Comment:
Revisit this test.  You are creating a valid PostImpression, but you wish to test for invalid Id's

Reply:
Done